### PR TITLE
label mgmt + ManagedClusterSets API

### PR DESCRIPTION
**Pull request by vMaroon**
- Created new bundle (& key) for managed clusters metadata (currently only labels)
- Created key entries for ManagedClusterSet, ManagedClusterSetBinding and Placement
- Moved apps, subscriptions and channels msg keys to this repo

### CODE REVIEW

File: bundle/spec/managed_cluster_labels.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package spec
+
+import (
+	"time"
+)
+
+// ManagedClusterLabelsSpec struct holds information for managed cluster metadata.
+type ManagedClusterLabelsSpec struct {
```
Comment (nirrozenbaum) [inline bundle/spec/managed_cluster_labels.go:8]: put in the same file as the bundle

File: msg_types.go [hunk 2]
```diff
@@ -20,6 +26,16 @@ const (
 	PolicyPlacementMsgKey = "PolicyPlacement"
 	// MinimalPolicyComplianceMsgKey - minimal policy compliance message key.
 	MinimalPolicyComplianceMsgKey = "MinimalPolicyCompliance"
+	// ControlInfoMsgKey - control info message key.
+	ControlInfoMsgKey = "ControlInfo"
```
Comment (nirrozenbaum) [inline msg_types.go:30]: no need for the keys that are not used in LH. 
for these we can just define the string in the HoH side.

File: msg_types.go [hunk 3]
```diff
@@ -5,11 +5,12 @@ const (
 	SpecBundle = "SpecBundle"
 	// StatusBundle - status bundle message type.
 	StatusBundle = "StatusBundle"
-	// Config - hub of hubs config message type.
-	Config = "Config"
 
 	// ManagedClustersMsgKey - managed clusters message key.
 	ManagedClustersMsgKey = "ManagedClusters"
+	// ManagedClustersMetadataMsgKey - managed clusters metadata message key.
+	ManagedClustersMetadataMsgKey = "ManagedClustersMetadata"
```
Comment (nirrozenbaum) [inline msg_types.go:12]: rename to ManagedClustersLabelsMsgKey
```
ManagedClustersLabelsMsgKey = "ManagedClustersLabels"
```

**Review by nirrozenbaum — COMMENTED** [review]
last comment (very minor)

File: msg_types.go (hunk 3 above)
Comment (vMaroon) [reply msg_types.go]: done

**Review by nirrozenbaum — APPROVED** [review]
/lgtm

**Pull request merged**